exasol/slc/api/run_db_tests.py

```diff
@@ -146,6 +147,8 @@ def run_db_test(
             raise api_errors.MissingArgumentError("external_exasol_db_port")
         if external_exasol_bucketfs_port is None:
             raise api_errors.MissingArgumentError("external_exasol_bucketfs_port")
+        if external_exasol_ssh_port is None:
+            raise api_errors.MissingArgumentError("external_exasol_ssh_port")
```

Comment:
Stumbled over this when running the new integration tests using an external db for "run_db_tests()"